### PR TITLE
Add log viewer for monitor

### PR DESCRIPTION
**Pull request by tei-k**
# Description
https://scalar-labs.atlassian.net/browse/DLT-7695

# Done
- Add [Tailon](https://hub.docker.com/r/port22/tailon)
 
Screen Shot 2020-12-17 at 16 07 46

### CODE REVIEW

File: modules/universal/monitor/provision/tailon/config.toml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+title = "Tailon file viewer"
+relative-root = "/"
+listen-addr = ":8088"
+allow-download = true
+allow-commands = ["tail"]
```
Comment (tei-k) [inline modules/universal/monitor/provision/tailon/config.toml:5]: custom config does not working. 🤔 
https://github.com/gvalkov/tailon/blob/fffdf3fe57cef675341c44cbee946d8a81c3ff8a/main.go#L230

File: modules/universal/monitor/provision/docker-compose.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-version: "3.1"
+version: "3.7"
```
Comment (feeblefakie) [inline modules/universal/monitor/provision/docker-compose.yml:1]: Is this a required change? or just upgraded?

**Review by feeblefakie — APPROVED** [review]
LGTM!
I think upgrading versions should be separately done usually unless it is related to the changes.

File: modules/universal/monitor/provision/docker-compose.yml (hunk 2 above)
Comment (tei-k) [reply modules/universal/monitor/provision/docker-compose.yml]: Just upgraded. Reveted in 648cfbd

**Review by ymorimo — APPROVED** [review]
LGTM! Nice!

**Pull request merged**